android/app/src/main/java/com/reactnativenavigation/views/LightBox.java

```diff
@@ -54,6 +54,15 @@ private void createContent(final Context context, LightBoxParams params) {
         lightBox.setBackgroundColor(params.backgroundColor.getColor());
         lightBox.addView(content, lp);
 
+        if (params.tapBackgroundToDismiss) {
+            lightBox.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    hide();
```

Comment:
I don't think so. Once the animation ends, the Dialog is dismissed and the onDismiss listener is called. How is `destroy` getting called?